test/dummy/config/environment.rb

```diff
@@ -1,3 +1,8 @@
+ENV["SHOW_DEVELOPER_AUTH"] = "true"
+ENV["GITHUB_APP_ID"] = "1234"
+ENV["GITHUB_APP_SECRET"] = "1234"
+ENV["ORGANIZATION_LOGIN"] = "my_org"
```

Comment:
oddly, the dummy app for tests runs in development mode and we need to set the env variable for it somewhere like here